charts/vouch/templates/_helpers.tpl

```diff
@@ -37,8 +37,8 @@ Common labels
 {{- define "vouch.labels" -}}
 helm.sh/chart: {{ include "vouch.chart" . }}
 {{ include "vouch.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- if or .Chart.AppVersion .Values.image.tag }}
+app.kubernetes.io/version: {{ .Values.image.tag | default .Chart.AppVersion | quote }}
```

Comment:
So i don't think this should be the image tag. Image tag can be latest@sha:... while thats not valid in the annotation (i learned from another repo)

Reply:
Good point. The issue with passing in a SHA-based tag is the 63 character limit for Kubernetes labels and the invalid characters. Added some handling for this case.